test/e2e/lib/flows/create-site-flow.js

```diff
@@ -6,6 +6,7 @@ import { By } from 'selenium-webdriver';
 /**
  * Internal dependencies
  */
+import config from 'config';
```

Comment:
nit: Technically an "external dependency", but pretty sure we have a ton of misplaced dependencies anyway.